templates/index.html

```diff
@@ -38,184 +43,263 @@ <h1 dir="ltr">IATI Data Store CSV Query Builder (Alpha Version)</h1>
             </div>
 
-            <div class="col">
-              <label for="entry_1414120858">
-                <div>Choose sample size
-                  <label for="itemView.getDomIdToLabel()" aria-label="(Required field)"></label>
+            <div class="row form-group">
+              <label for="entry_18398991">
+                <div>
+                  <a href="#" class="tooltip-ui" data-toggle="tooltip" title="This allows you to filter by the type of organisation. To access data on NGOs only you would select International NGO and/or National NGO and/or Regional NGO.">
+                    Select type of reporting organisation (eg. INGO = 21)
+                  </a>
                 </div>
               </label>
-              <ul>
-                <li>
-                  <label>
-                    <input type="radio" name="entry.1352830161" value="50 rows" id="group_1352830161_1" aria-label="50 rows"
-                    {% if not size or (size and size == "50 rows") %}
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>50 rows</span>
-                  </label>
-                </li>
-                <li>
-                  <label>
-                    <input type="radio" name="entry.1352830161" value="Entire selection" id="group_1352830161_2" aria-label="Entire selection"
-                    {% if size and size == "stream=True" %}
-                      checked="checked"
-                    {% endif %}
-                    />
-                    <span>Entire selection</span>
-                  </label>
-                </li>
-              </ul>
+              <div class="clearfix"></div>
+              <select data-placeholder="Choose a reporting organisation type..." multiple class="chosen-select form-control" id="entry_18398991[]" name="entry.18398991[]">
+                {{ include('include/reporting_org_type.html') }}
+              </select>
             </div>
           </div>
 
-          <div class="row form-group">
-            <label for="entry_1922375458">
-              <div>Select reporting organisation (eg UK DFID = GB-GOV-1)</div>
-              <div dir="ltr"></div>
-            </label>
+          <div class="step-section">
+            <div class="title">
+              <span class="step">2</span>
+              <h3>Select the sector</h3>
+            </div>
             <div class="clearfix"></div>
-            <select data-placeholder="Choose a reporting organisation..." multiple class="chosen-select form-control" id="entry_1922375458[]" name="entry.1922375458[]">
-              {{ include('include/reporting_org.html') }}
-            </select>
+
+            <div class="row form-group">
+              <label for="entry_1954968791">
+                <div>
+                  <a href="#" class="tooltip-ui" data-toggle="tooltip" title="Choose the sector or sectors you are looking for.">
```

Comment:
Suggest changing from `<a>` elements to `<span>` elements so that clicking on them doesn't jump you around the page. This is particularly important for the radio buttons where selecting the label should select that radio option.

Reply:
Good idea!